Core/templates/blog.html

```diff
@@ -1,41 +0,0 @@
-{% extends 'base.html' %} {% block blog_list %}
```

Comment:
Why have you deleted this? Are we adding another template for the blogs?

Reply:
Sir will add a new template it will reflect in changes you requested